csrc/fusion.h

```diff
@@ -146,8 +148,11 @@ class TORCH_CUDA_CU_API Fusion : public IrContainer {
   bool isNoOp();
 
   //! Lower the fusion and evaluate bank conflict info
-  std::unordered_map<std::string, std::pair<int, int>> bankConflictInfo(
-      const CompileParams& compile_params = CompileParams());
+  //! Returns (tensor, read conflict ways, write conflict ways)
+  //! Each tensor can be read/write by multiple expressions, so the ways are
+  //! vectors.
+  std::unordered_map<TensorView*, std::pair<std::vector<int>, std::vector<int>>>
```

Comment:
Now I have a mechanism to associate bank conflict info with tensors in the fusion, so it no longer needs to return string as key anymore.